### PR TITLE
Fixing help message and improving error handling

### PR DESCRIPTION
**Pull request by tommyg-cld**
#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [ ] New feature
- [x] Bug fix
- [ ] Adds more tests

#### Are tests included?
- [ ] Yes
- [x] No

#### Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

File: cloudinary_cli/modules/regen_derived.py [hunk 1]
```diff
@@ -25,7 +24,7 @@
 @option("-A", "--auto_paginate", is_flag=True, default=False,
         help="Auto-paginate Admin API calls.")
 @option("-F", "--force", is_flag=True,
-        help="Skip initial and auto-paginate confirmation.")
+        help="Skip initial confirmation.")
```
Comment (const-cloudinary) [inline cloudinary_cli/modules/regen_derived.py:27]: Maybe change to `Skip confirmation.` or `Skip user confirmation.`. like, `initial` is kind of ambiguous, no context.
@carlevison, WDYT?

File: cloudinary_cli/modules/regen_derived.py [hunk 2]
```diff
@@ -63,6 +62,9 @@ def regen_derived(trans_str, eager_notification_url,
                                        api_name="admin",
                                        auto_paginate=auto_paginate,
                                        force=force, return_data=True)
+    if trans_details is False:
+        exit()
```
Comment (const-cloudinary) [inline cloudinary_cli/modules/regen_derived.py:66]: Maybe write an error message saying what went wrong? Just exiting can be very confusing. And it you run it in script/cron job - you have no idea what happened/ did not happen.

**Review by const-cloudinary — CHANGES_REQUESTED** [review]

File: cloudinary_cli/modules/regen_derived.py (hunk 1 above)
Comment (carlevison) [reply cloudinary_cli/modules/regen_derived.py:27]: @const-cloudinary 'Skip confirmation.' is fine.

Comment (tommyg-cld) [reply cloudinary_cli/modules/regen_derived.py]: updated to `Skip confirmation.`

File: cloudinary_cli/modules/regen_derived.py (hunk 2 above)
Comment (tommyg-cld) [reply cloudinary_cli/modules/regen_derived.py]: as discussed I've updated this to `return False` instead

**Comment by const-cloudinary** [comment]
@tommyg-cld great PR!

**Pull request merged**